compile/unminified/pm-image-editor.js

```diff
@@ -198,10 +198,11 @@
             },
             link: function (scope, element) {
                 var buttonNames = 'crop,rotate-cw,rotate-acw,flip-h,flip-v,undo,redo';
+                var buttonLabels = ['Crop', 'Rotate Clockwise', 'Rotate Counterclockwise', 'Flip Horizontal', 'Flip Vertical', 'Undo', 'Redo'];
                 var buttons = {};
-                buttonNames.split(',').forEach(function(name){
+                buttonNames.split(',').forEach(function(name, index){
                     buttons[name] = angular
-                        .element('<span class="image-editor-'+name+'" />')
+                        .element('<span class="image-editor-'+name+'" title="'+buttonLabels[index]+'" />')
```

Comment:
Might be safer to do `window.escape(buttonLabels[index])` so that if future label translations include special characters, the HTML doesn't become invalid.

(Sorry, I'm still subscribed to this repository!)